ansible/roles/datadog/templates/process.yaml.j2

```diff
@@ -4,37 +4,56 @@
 init_config:
 
 instances:
+{% if datadog_integration_celery %}
```

Comment:
interesting, i had assumed it would pretty much be a noop if the process didn't exist on that server. did you find otherwise?